docs/_articles/en/lwc/lwc-get-started.md

```diff
@@ -0,0 +1,100 @@
+
+
+```
+public with sharing class ContactController {
+    @AuraEnabled(cacheable=true)
+    public static List<Contact> getContacts() {
+        return [
+SELECT 
+Id, 
+Name, 
+Email, 
+Phone 
+FROM Contact 
+WITH SECURITY_ENFORCED 
+LIMIT 10
+];
+    }
+}
+
+```
+Next we'll add the LWC component that displays queried fields.
+
+### Create an LWC Component that Displays Contact Information
+1. From the command palette run **SFDX: Create Lightning Web Component** and create a component called `contactList` in the default location.
+2. In the `contactList.html file`, cut and paste the following code and save the file:
+
+```
+<template>
+    <lightning-card title="Contact List">
+        <template if:true={contacts.data}>
+            <template for:each={contacts.data} for:item="contact">
+                <div class="slds-var-p-horizontal_medium" key={contact.Id}>
+                    <p>{contact.Name}, {contact.Phone}</p>
+                </div>
+            </template>
+        </template>
+        <template if:true={contacts.error}>
+            <p>{contacts.error}</p>
+        </template>
+    </lightning-card>
+</template>
+
+```
+3. In the contactList.js file, cut and paste this code and save:
+
+```
+import { LightningElement, wire } from 'lwc';
+import getContacts from '@salesforce/apex/ContactController.getContacts';
+
+export default class ContactList extends LightningElement {
+    @wire(getContacts) contacts;
+}
+```
+4. Cut and paste the following code in the contactList.js-meta.xml file and save:
+   
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+ 	<apiVersion>57.0</apiVersion>
+ 	<isExposed>true</isExposed>
+ 	<targets>
+
+   		<target>lightning__HomePage</target>
+ 	</targets>
+</LightningComponentBundle>
+  
+```
+5. Right-click the default folder under `force-app/main` and run **SFDX: Deploy Source to Org** to deploy your code to the org.
+
+### Add the New Component to Your App in Lightning Experience
+1. In Visual Studio Code, open the Command Palette and run **SFDX: Open Default Org**.
+2. This opens your org in a separate browser.
+3. From the App Launcher (App Launcher), find and select **Sales**.
+4. Click **Setup** gear then select Edit Page.
+5. Drag the `contactList` Lightning web component from the Custom area of the Lightning Components list to the top of the Page Canvas. 
+6. Click **Save**.
+7. Click **Activate**.
+8. Click **Assign as Org Default**.
+9. Click **Save**.
+10. Click **Save** again, then click **Back** arrow to return to the page.
+11. Refresh the page to view your new component.
+
+![PNG showing LWC component](../../images/../../images/contact_lwc.png)
```

Comment:
What do you think about doing a Part Two to this, where the user then retrieves the new changes from the org to their IDE and closes the loop?

Reply:
Good idea. I can add that in the next iteration.